wrlinux/wrlinux_test.go

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2022 Wind River Systems, Inc.
+ *
+ * The right to copy, distribute, modify, or otherwise make use
+ * of this software may be licensed only pursuant to the terms
+ * of an applicable Wind River license agreement.
+ */
+
+package wrlinux
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_parse(t *testing.T) {
+	type args struct {
+		filePath string
+	}
```

Comment:
I thinks we can move `filePath` to `testCases` struct. 
Looks liike we don't need `args` struct.